server/chat-commands/admin.ts

```diff
@@ -104,6 +104,25 @@ async function rebuild(context: CommandContext) {
 
 
 export const commands: ChatCommands = {
+	potd(target, room, user) {
+		this.canUseConsole();
+		target = toID(target);
+		const species = Dex.getSpecies(target);
+		if (!species.exists) return this.errorReply(`Pokemon ${target} not found.`);
+		if (!Dex.getLearnsetData(species.id).learnset) {
+			return this.errorReply(`That Pokemon has no learnset and cannot be used as the PotD.`);
+		}
+		// this _could_ be an instanceof check but worst case we send it to another PM
+		// that's in that file and it does nothing
+		const simProcesses = ProcessManager.processManagers.filter(p => p.basename.includes('room-battle'));
+		for (const sim of simProcesses) {
+			for (const sub of sim.processes) {
+				sub.getProcess().send(`EVAL\n\nConfig.potd = '${species.id}'`);
+			}
+		}
```

Comment:
Why are you manually searching through PMs? Isn't this just `Rooms.PM`?

Reply:
honestly tho why is this `Rooms.PM` - why not `Rooms.RoomBattle.PM`? It's confusing.